content/expressions/methods.md

```diff
@@ -10,7 +10,7 @@ toc: true
 
 All Pony code that actually does something, rather than defining types etc, appears in named blocks which are referred to as methods. There are three kinds of methods: functions, constructors, and behaviours. All methods are attached to type definitions (e.g. classes) - there are no global functions.
 
-Behaviours are used for handling asynchronous messages sent to actors. We'll look at those later.
+Behaviours are used for handling asynchronous messages sent to actors, which we've seen in the "Types" chapter [when we talked about actors]({{< relref "types/actors.md" >}}).
```

Comment:
does relref allowing linking directly to `#behaviours`?

If yes, then I think it should do that. Otherwise this can be merged.